Automattic/jetpack
# Add activation status and change position of licensing actions in My Plans section

Asana card: 1201509629272450-as-1201545797263879

**Changes proposed in this Pull Request:**

This PR implements the following enhancements:

- Add the "Activated" label to the recently activated plan or product.
- Display the licensing actions to the top of the section when product licenses are not attached to any site yet. If there are no "unattached" licenses available, the container should be displayed at the bottom (as it was before).

**Does this pull request to change what data or activity we track or use?**

- No.

**Testing instructions:**

Case 1: Testing the "Activated" label

- Set up a new self-hosted site with this branch as the plugin version.
- Activate Jetpack and purchase any available product, make sure you skip the site selection to get the license key.
- Visit wp-admin/admin.php?page=jetpack#/license/activation and activate your product.
- Visit wp-admin/admin.php?page=jetpack#/my-plan
- Verify that the "Activated" label is displayed next to your product.
- Verify that the UI is looking good on different screen resolutions.

Case 2: Testing the position of the licensing actions

Note: If you have any abandoned licenses, you can try the test below with a new WordPress.com account.

- Set up a new self-hosted site with this branch as the plugin version
- Activate Jetpack and purchase any available product, make sure you skip the site selection to get the license key. Don't activate the purchased product yet.
- Visit wp-admin/admin.php?page=jetpack#/my-plan
- Verify that the licensing actions are displayed on the top.
- Visit wp-admin/admin.php?page=jetpack#/license/activation and activate your product.
- Visit wp-admin/admin.php?page=jetpack#/my-plan
Verify that the licensing actions are displayed on the bottom if no "unattached" licenses are available.
- Verify that the UI is looking good on different screen resolutions.

**Screenshots:**

Before | After
--- | ---
![Screenshot 2021-12-22 at 18 06 17](https://user-images.githubusercontent.com/1749918/147122894-05d2135f-c67c-4527-ba08-7a0964b95c0a.png) | ![Screenshot 2021-12-22 at 17 49 25](https://user-images.githubusercontent.com/1749918/147122915-cd0e3a44-997e-49c4-9092-7b5f5ed32330.png)
